homeassistant/generated/integrations.json

```diff
@@ -5512,6 +5512,12 @@
       "config_flow": true,
       "iot_class": "local_push"
     },
+    "switchbot_cloud": {
```

Comment:
By the way, I think it's good to make a Switchbot brand and then put switchbot and switchbot cloud underneath

Reply:
@joostlek Do you have some documentation or an example about how to do this ?

Reply:
I'm on mobile, but search for weatherkit, they have an open PR where they add one for apple

Reply:
Thanks ! I will check it tomorrow.

Reply:
@joostlek I am so sorry, I missed the documentation yesterday.

Reply:
wdym?

Reply:
@joostlek I asked you yesterday "Do you have some documentation or an example about how to do this ?" but it was clearly documented lol

Reply:
Aaah good :)